src/main/java/red/mohist/Mohist.java

```diff
@@ -25,14 +26,9 @@ public static String getVersion() {
   }
 
   public static void main(String[] args) throws Throwable {
-    if(Float.parseFloat(System.getProperty("java.class.version")) != 52.0) {
-      System.out.println(Message.getString("unsupported.java.version"));
-      System.exit(0);
-    }
-    if(System.getProperty("log4j.configurationFile") == null) {
-      System.setProperty("log4j.configurationFile", "log4j2_mohist.xml");
-    }
-      System.out.println("\n" +
+    if(Float.parseFloat(System.getProperty("java.class.version")) != 52.0 || bMohist("use_custom_java8", "false"))
```

Comment:
```suggestion
    if(Float.parseFloat(System.getProperty("java.class.version")) != 52.0 || bMohist("use_custom_java8"))
```
There's no point of using a defaultReturn. It is already false.

Reply:
Resolved